oasislmf/model_preparation/summaries.py

```diff
@@ -717,23 +718,41 @@ def write_exposure_summary(
     :return: Exposure summary file path
     :rtype: str
     """
+    loc_per_cov = oed_hierarchy['locperilid']['ProfileElementName'].lower()
+    exposure_df = reduce_df(
+        exposure_df,
+        cols=[loc_per_cov, 'loc_id']
+    )
+    gul_inputs_df = reduce_df(
+        gul_inputs_df,
+        cols=['peril_id', 'coverage_type_id', 'loc_id', 'tiv']
+    )
```

Comment:
Could also be achieved with:
```
gul_inputs_df = gul_inputs_df[['peril_id', 'coverage_type_id', 'loc_id', 'tiv']]
```
or:
```
gul_inputs_df = gul_inputs_df.loc[:, ['peril_id', 'coverage_type_id', 'loc_id', 'tiv']]
```
However, separate function does look tidier, so no objection.